test/mocks/server/transport_socket_factory_context.h

```diff
@@ -22,7 +23,9 @@ class MockTransportSocketFactoryContext : public TransportSocketFactoryContext {
   MockTransportSocketFactoryContext();
   ~MockTransportSocketFactoryContext() override;
 
-  Secret::SecretManager& secretManager() override { return *(secret_manager_.get()); }
+  Secret::SecretManager& secretManager() override { return *secret_manager_; }
+  // Secret::MockSecretManager& secretManager() override { return secret_manager_; };
```

Comment:
Can you remove the commented out code?

Reply:
removed these tests and rather added integration test in the oauth as you suggested.